suite-native/intl/src/en.ts

```diff
@@ -50,6 +53,81 @@ export const en = {
             hodlOn: 'Hodl on tight',
         },
     },
+    moduleOnboarding: {
+        welcomeScreen: {
+            welcome: 'Welcome to ',
+            subtitle: 'Securely track, manage & receive crypto on the go ',
+            trezorLink: {
+                text: 'Don’t have a Trezor? <trezorLink>Get one here.</trezorLink>',
+                url: 'https://trezor.io/',
+            },
+            nextButton: 'Get started',
+        },
+        connectTrezorScreen: {
+            title: 'Connect',
+            subtitle:
+                'Manage your portfolio with your Trezor hardware wallet connected directly to your mobile device.',
+            buttons: {
```

Comment:
I don't think we need to define these generic button texts again and again, we should have just one translation for this module or even for whole app.

Reply:
Done